modules/document_repository/README.md

```diff
@@ -0,0 +1,49 @@
+# Document Repository
+
+## Purpose
+
+The Document Repository is a tool that provides a centralized location for non-candidate documents relevant to the study. For example, copies of non-filled instruments, publications, manual of operations.
+
+## Intended Users
+
+ LORIS users can view, download/upload files, edit information about files, and delete files given appropriate permissions.
+This module is used primarily by study coordinators/administrators who share study protocols and manuals, and by data entry staff who consult these documents.  
+
+## Scope
+
+ The document can only be seen by Users from the Site that the document is associated with.
+
+## Permissions
+
+ A user that has "document_repository_view" permission can view and upload files in Document Repository.
+ A user that has "document_repository_delete" permission can delete files in Document Repository.
+
+## Configurations
+
+
+- The Document Repository enables users to upload and organize project files of any type that can easily be viewable for users with appropriate permissions. Give full permissions to store documents on server.
+
+- To enable file's upload: please update the php.ini apache configuration file.
+```
+session.gc_maxlifetime = 10800  // After this number of seconds, stored data will be seen as 'garbage' and cleaned up by the garbage collection process.
+max_input_time = 10800          // Maximum amount of time each script may spend parsing request data (in seconds)
+max_execution_time = 10800      // Maximum execution time of each script (in seconds)
+upload_max_filesize = 1024M     // Maximum allowed size for uploaded files.
+post_max_size = 1024M           // Maximum size of POST data that PHP will accept.
+```
+
+- To enable folder upload:
+```
+chmod 777 /var/www/loris/modules/document_repository/user_uploads
+
+```
+Note that the path is assumed to be /var/www/loris however your own path may be /var/www/<project-name>, depending on your setup.
```

Comment:
Sorry, I missed this in my first series of comment. The path should probably be in code format, no? That would fix the issue that we don't see `<project-name>` in the display of the MD file.
Same for `php.ini` in line 26.

Reply:
Would also have a comma after the `/var/www/loris` path so that it looks like this:

_Note that the path is assumed to be `/var/www/loris`, however your own path may be `/var/www/<project-name>`, depending on your setup._

This "note" sentence should probably be between brackets or in italic so that it is clear that the 3 following items belong to the Configuration section and are not items belonging to the note (I got confused for a little while when reading it)